moveit_ros/planning/planning_request_adapter_plugins/src/fix_start_state_bounds.cpp

```diff
@@ -181,34 +180,14 @@ class FixStartStateBounds : public planning_request_adapter::PlanningRequestAdap
       }
     }
 
-    bool solved;
     // if we made any changes, use them
     if (change_req)
     {
-      planning_interface::MotionPlanRequest req2 = req;
-      moveit::core::robotStateToRobotStateMsg(start_state, req2.start_state);
-      solved = planner(planning_scene, req2, res);
+      RCLCPP_WARN(logger_, "Changing start state.");
+      moveit::core::robotStateToRobotStateMsg(start_state, req.start_state);
     }
-    else
-      solved = planner(planning_scene, req, res);
 
-    // re-add the prefix state, if it was constructed
```

Comment:
without the block below, this adapter is somewhat useless (not that I'm arguing that it is overwhelmingly useful otherwise). I would suggest replacing these "hacks" with validity checks that throw an error if the precondition is not met - here start state out of bounds -> fail and tell the user

Reply:
I updated all the adapters.